scripts/testing_scripts/test_all_sandia

```diff
@@ -636,6 +636,7 @@ fi
 export OMP_NUM_THREADS=8
 export OMP_PROC_BIND=spread
 export OMP_PLACES=cores
+export OMP_MAX_ACTIVE_LEVELS=1
```

Comment:
Should we be setting it to `3`?  @Rombur ?

Reply:
I picked `3` for no good reason. The behavior is different if it is one or if it's greater than one. When the max level is one, you do not allow for nested parallelism and so, you kind of skip the code in `partition_master`. I think it would be useful to try with a number greater than one and see what happens.

Reply:
@dalg24 @Rombur  I tested with gcc/6.4.0 and OpenMP backend enabled, the partition_master tests fail with 3 set as the max level, but pass with 1

Reply:
OK then let's just use 1